fix(user type): add user type table (#11550)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx

  <tbody>
    <tr>
      <td>

>         Basic users are free. Basic users can set up our observability tools, run queries of your data, use custom dashboards, use some basic alerting features, and more. They **can't** use our curated experiences (for example, our APM UI, our browser UI, or our mobile UI).

Reviewer: ```suggestion
        Basic users are free. Basic users can set up our observability tools, run queries of your data, use custom dashboards, use some basic alerting features, and more. They **can't** use our curated experiences (for example, our APM UI, browser UI, or mobile UI).
```

**Reasons to make someone a basic user:**

> * They don't need to use our curated experiences and dashboards, but would benefit from the ability to create their own custom queries and charts of data; in other words, they don't need full access to the platform.

Reviewer: ```suggestion
* They don't need full platform access to use our curated experiences and dashboards, but would benefit from creating custom queries and charts of data.
```

* They play a key role in the planning phase of the application development lifecycle.
* They use and configure New Relic tools to get data into New Relic, and access, configure, and use alerts on such data, but are not necessarily responsible for triaging workflows, troubleshooting, or managing users and roles for their team.
* They want to see high-level analytics and business metrics for future planning (as is often true of C-suite executives).
* They don't manage users or billing.

## User type capability comparison table [#user-type-comparison-table]

Below is a detailed comparison of capabilities by user type. Important points about this table:

> * The table comes from [our pricing page](https://newrelic.com/pricing). To find the table: go to the **User costs** heading and click **View permissions**. 

Reviewer: ```suggestion
* The table comes from [our pricing page](https://newrelic.com/pricing). To find the table, visit the **User costs** heading and click **View permissions**. 
```

> * Many of the features refer to access to our UI experiences, not to the underlying data. All users, including basic users, can query all data in the accounts they have access to, and can create and view custom charts. Put another way: basic users can access APM data, browser monitoring data, and more, but they don't have access to our curated UI experiences. 

Reviewer: ```suggestion
* Many of the features require access to our UI experiences, not to the underlying data. All users can query all data in the accounts they can access and can create and view custom charts. For instance, basic users can access APM data, browser monitoring data, and more, but can't access curated UI experiences. 
```

      </tbody>
    </table>
  </Collapser>
  <Collapser
    id="api-access"
    title="APIs: data ingest and NerdGraph"
  >
Details about access to [our APIs](/docs/apis/intro-apis/introduction-new-relic-apis): 

> * **Data ingest**: All user types can set up almost all of our integrations and agents, including ingesting data via our ingest APIs. There are some New Relic solutions that require being a full platform user to set up: if that's the case, it will be mentioned in the requirements section for the docs of those solutions. 

Reviewer: ```suggestion
* **Data ingest**: All user types can set up almost all of our integrations and agents, including ingesting data via our ingest APIs. Some New Relic solutions require being a full platform user to set up. If so, it is mentioned in the requirements section for the docs of those solutions. 
```

* **NerdGraph**: A user can use [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to do the same things they can do from the UI. For example: configuring APM settings requires being a full platform user, whether that configuration is done via the UI or via NerdGraph. For another example: the [administration-related capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#admin-settings) (for example, adding users, and adding accounts) require being a core user or full platform user, whether that's done in the UI or via NerdGraph.

> If you get an error message when using an API that says you don't have permissions, it may be related to user type restrictions. For more on access factors, see [Factors affecting access](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data).  

Reviewer: ```suggestion
Error messages when using an API about not having permissions may relate to user type restrictions. For more on access factors, see [Factors affecting access](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data).  
```

</Collapser>

  <Collapser
    id="quickstarts"
    title="Instant Observability quickstarts"
  >
Details about access to the options available on [our Instant Observability page](https://newrelic.com/instant-observability): 

> * Basic and core users can install almost all New Relic solutions but, as shown in the user type table, can't access the curated views (for example, APM views, mobile monitoring views, and other experiences).

Reviewer: link to user type table?
